docs/contrib/PlatformBringUp.md

```diff
@@ -285,14 +270,14 @@ set_routing_layers -signal $::env(MIN_ROUTING_LAYER)-$::env(MAX_ROUTING_LAYER)
 ```
 
 
-The first command, set_global_routing_layer_adjustment, adjusts the routing resources of the design. This effectively reduces the number of routing tracks that the global router assumes to exist. By setting it to the value of 0.5, this reduced the routing resources of all routing layers to 50% which can help with congestion and reduce the challenges for detail routing. The second command, set_routing_layers, sets the minimum and maximum routing layers for signal nets by using the -signal option.
+The first command, `set_global_routing_layer_adjustment`, adjusts the routing resources of the design. This effectively reduces the number of routing tracks that the global router assumes to exist. By setting it to the value of 0.5, this reduced the routing resources of all routing layers to 50% which can help with congestion and reduce the challenges for detail routing. The second command, `set_routing_layers`, sets the minimum and maximum routing layers for signal nets by using the `-signal` option.
 
-More customization can be done to increase the efficiency of global and detail route. Refer to the [OpenROAD documentation](https://openroad.readthedocs.io/_/downloads/en/latest/pdf/).
+More customization can be done to increase the efficiency of global and detail route. Refer to the [FastRoute documentation](https://openroad.readthedocs.io/en/latest/main/src/grt/README.html)
```

Comment:
this should be a relative link to the grt README.md

Reply:
@rjridle note that due to some quirks of rtd the links should be `./main/src/grt/README.md`

Reply:
Changed to relative link.